lib/jbuilder/jbuilder_template.rb

```diff
@@ -170,8 +171,8 @@ def _render_partial_with_options(options)
   end
 
   def _render_partial(options)
-    options[:locals].merge! json: self
-    @context.render options
+    options[:locals][:json] = self
+    @context.render options, nil
```

Comment:
The `render` helper in rails will default the second parameter to `{}`. By providing `nil` here we save on that extra memory allocation.

That second parameter is intended to be the options you provide to the partial if the first param is the partial name (ex: `render 'foo', options`). Since the partial name is included in the `options`, that second parameter isn't actually used.